spec/presenter_spec.rb

```diff
@@ -2,10 +2,25 @@
 require "tictactoe"
 
 RSpec.describe Presenter do
-  describe "#display_board" do
+  subject(:presenter) { Presenter.new }
+
+  class Presenter::FakeBoard
+    def initialize
+      @board = Array.new(9)
+    end
+
+    def place_token(position)
+      @board[position - 1] = "X"
+    end
```

Comment:
It seems like we only need `place_token` in this fake in order to put the board in the right state before we present it. An alternative could be to do that with the initializer. Just a suggestion.

Reply:
I'll look into that suggestion.